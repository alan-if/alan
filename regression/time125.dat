You can't take people with you!You can't take people with you!You can't open people.You can't close people.You can't examine that.You can't tie that.You can't cut that.You can't read people.You don't expect a response, do you?You don't expect a response, do you?You don't expect a response, do you?You don't expect a response, do you?You don't expect a response, do you?You don't expect a response, do you?You can't throw people about. It isn't nice.You can't throw people about. It isn't nice.You can't throw anything at that.You can't throw people about. It isn't nice.You can't throw anything at that.You can't throw people about. It isn't nice.You would rather not throw that.You should not attack the $o.You should not attack the $o.You can't open that.You can't close anything with that.You can't close that.You can't close anything with that.You can't lock that.You can't lock that.You can't eat that.You can't drink that.You can't put that anywhere.You can't put that.You can't put anything with the $2.You can't put that.You can't put anything near the $2.You can't put that.You can't put anything behind the $2.You can't put that.You can't put anything on the $2.You can't put that.You can't put anything under the $2.You can't use that.You can't push that.You can't push that.You can't use a $2 to push anything.You can't turn that on.You can't turn that off.You can't pull that.You can't play that.You can't play with that.You can't light that.You can't extinguish that.You can't put that out.You can't turn that off.You can't turn that off.You can't shoot that.You can't turn that off.You can't shoot that.You can't wear that.You can't put that on.You can't remove that.You can't take that off.There is no score used in this game.You let out a primal scream, which accomplishes nothing. Although you do feel a little better.Time passes...Saving...Done.Restoring...Done.$nSorry, but you're on your own.You haven't got that.You would rather hold onto it for now.You can't take that!You've already got that.Taken.You can not use that.You use the $o.You can not cut that.You cut the $o.You can't open that!It's already open.The $o is now open.You can't close that.It is not open.The $o is now closed.There is nothing special about the $o.There is nothing special inside the $o.There is nothing special under the $o.There's nothing special about the $o.Don't lick that. You don't know where it has been.There's nothing to read here.The $o ignores your question.$o ignores your question.The $1 ignores your question.$1 ignores your question.The $o is not impressed.$o is not impressed.The $1 is not impressed.$1 is not impressed.The $o does not seem to want to talk.$o does not seem to want to talk.That would not be healthy.The $1 does not seem to want to talk.$1 does not seem to want to talk.You haven't got that!I don't see any $2 here.The $1 bounces harmlessly off the $2.$nDropped.You would rather not throw the $1.You can't create a $o.You haven't got that!I don't see any $2 here.Now, that would be a good trick!Done.You would rather not drop the $1 at this time.I don't see that person here.There is no need for violence at this time.I don't see that person here.You don't have that object to attack with.There is no need for violence at this time.I don't see any $1 to open.You don't have the $2.You can't open the $1 with the $2.I don't see any $1 to close.You don't have the $2.You can't close the $1 with the $2.You can't lock that!I don't see what you wish to lock.It's already locked.The $o is now locked.You can't unlock that!I don't see what you wish to unlock.It's already unlocked.The $o is now unlocked.That is not edible.I don't see what you want to eat.You eat the $o. Yum, that was delicious!That is not drinkable.I don't see what you want to drink.You drink the $o. It is quite refreshing.You haven't got that!I don't see any $2 here.You can't do that.I don't see what you want to push.You can't push it.I don't see what you want to push.You don't have any $2.Your action has no noticeable effect.You can't turn that on.I don't see what you want to turn on.It's already on.Ok, the $o is now on.You can't turn that off.It's already off.I don't see what you want to turn off.Ok, the $o is now off.I don't see what you want to pull here.You can't pull that.I don't see what you want to play here.You can't play that.I don't see the object you want to light.You can't light that object.I don't see the object you want to extinguish.You can't extinguish that object.I don't see what you want to shoot.You would rather not do that.You can't do that.You don't have that object.I don't see what you want to shoot.You would rather not do that.You can't do that.You don't have that object.I don't see what you want to shoot.You can't do that.You don't have that object.You can't put that on.You don't have that object.You can't remove that.inventoryYou are carrying:You are carrying nothing.waterwaveswhite sandlost professorYou can't see the Professor from here.How would you like to signal him?You can't do that.You can't signal him with the $o.You don't have any metal.You aim the metal at the sun, trying to create a signal that the Professor can see. The shiny metal reflects the sunlight quite well. After about ten minutes, you start to see a similar signal in the far distance, across the river along the shoreline. It must be the Professor. You now have an approximate idea of where he is.You can't signal the Professor with that.large treeYou climb down the tree.rocky trailforestrocky trailforestrocky traildark forestgreen leafrocky traillarge hillYou climb up the hillside.large hillYou climb down the hill.small cavedark smokeIt doesn't appear to be coming from a natural fire.Your lungs are not quite powerful enough to suck the smoke in from this distance. I'm not really sure why you would want to anyway.protruding rockIt looks very unstable.dark cavedark tunnelwaterfallrivercoastlineYou are too far away to make out any details.large tyranosaurusThat would be suicide.You decide to examine the dinosaur at a much closer distance by entering his body. The Tyranosaurus swallows you in one bite, killing you instantly.small dead dinosaurThat may be your fate if you are not careful.smokeIt looks like it is coming from very nearby.You would rather not.strange dinosaurThat would be too dangerous.They are currently blocking your path in their direction.You would rather not do that.There is no need to do that.You do not have a dinosaur egg.large pitIt is too deep to safely cross.You can not do this and still survive.large treeIt is covered with vines.You climb up the large tree.dirty waterThe water is not very clear.You don't think this water would be safe to use.large treeThe tree is covered with vines.You climb down the tree.large pitIt is too deep to safely cross.You can not do this and still survive.large pitThere does not appear to be a way to cross the pit from here.You can not do this and still survive.large pitYou are located directly over it. It is quite large and deep.That would not be wise.You go far as you dare on the long branch. You then grab hold of the branch and swing yourself over the pit. You just manage to clear it, landing in a few bushes. You are a bit scratched, but nothing more.large treeYou climb up the tree.large nestThere are several large eggs located within it.shorelineYou can just make out the shoreline through the plant growth. The large dinosaurs here are blocking further passage to the shore.sturdy branchTo your direct east is a large, strudy branch.sturdy branchIt is long and quite sturdy. It extends out over the pit below.You go far as you dare on the long branch. You then grab hold of the branch and swing yourself over the pit. You just manage to clear it, landing in a few bushes. You are a bit scratched, but nothing more.blue waterIt is quite wet.You reach into the fast flowing water and wash off.large dinosaurYou are not sure if it is a dinosaur.herd of large dinosaursThey appear to have large beaks.That would be too dangerous.green bushesYou are not sure, but you think you can see something moving behind them.You can't see anything moving behind them any longer.green vineYou do not need any more vines.large green leafYou do not need any more leaves.dark sticky mudThe mud is quite thick and sticky.You don't need any more mud.dark sticky mudThe mud is quite thick and sticky.How would you like to use the mud?You would rather keep the mud for now.You don't have any mud.You already have all the mud you could possibly need.What would you like to put the mud in?You would rather not do that.You would rather not do that.You have all the mud you could possibly need.You fill the cans up with mud.You have nothing to hold the mud in.You can not do that.You rub the mud onto the vines and wood. You wait a short while for the mud to dry. It quickly does so, creating a stronger bond. You have no need to use the mud at this time.There is nothing here to use the mud on.You have no need to do that.electric lampIt is currently on.You would rather leave it here.You would rather leave it on.green vineYou already have them.They are too hard to break with your bare hands.You would rather hold onto the vines for now.You do not have them.You can't do that.You can't do that.You have nothing to cut them with.You already have them.You cut down the vines.You have no need for any more vines.What would you like to cut them with?The vines appear to be strong.How would you like to use them?What would you like to tie them to?You can not do that.You have no need to tie that.You tie the wood together using the vines. You now have a small raft.The wood is too large to tie up.There is nothing here to tie.large eggsThe eggs are a bit large for you to swallow.You can't hold any more eggs.You start to take a dinosaur egg. Just as you are picking it up, a large dinosaur pops up from behind the bushes. It must be the mother. She attacks you wildly and kills you.You quickly take a few of the large dinosaur eggs.The eggs are very large.There is no need to drop them here.piece of shiny metalpiece of rusted metalThe metal is quite shiny and reflective.The metal is a rusty and dirty.You would rather hold onto the metal for now.You are not holding the piece of metal.You clean off the piece of metal using the nearby water.There is nothing here to clean it with.You can't do that.You clean off the metal using the nearby water.You can't clean the piece of metal using that.You aim the metal at the sun, trying to create a signal that the Professor can see. The shiny metal reflects the sunlight quite well. After about ten minutes, you start to see a similar signal in the far distance, across the river along the shoreline. It must be the Professor. You now have an approximate idea of where he is.There is no need to do that here.There is an old iron box here.old iron boxIt is a large box. The hinges appear to be very rusty. This box is proof the Professor must have been here.The box is too heavy to easily carry around.It is already open.You manage to lift the rusty lid of the box. It seems that rust had been holding the box together, as it seems to fall into pieces as the oily substance flows over it. There are some basic supplies here. There is some canned food, a small axe and an electronic lamp. You set the lamp up, illuminating the cave.You try to open the lid, but it is stuck from rust.You would rather leave it open.It is already closed.You do not need to oil that.You can't do that.You put the oily substance on the hinges of the box.You have nothing to oil it with.You put the oily substance on the hinges of the box.You have nothing to oil it with.selection of canned foodDon't just lick it. Get a nice mouthful.You open the cans of food and dig in. The food isn't exactly delicious, but it is edible.few cansThey are already open.You can't close them.You do not need to fill that.You do not need to do that.You do not need any more mud.You fill the cans with mud.You fill the cans with the mud.You have no need to do this.You would rather hold onto the cans for now.There is an axe here.small axeIt is a small axe. The blade is very sharp, however.Hmm, it just might not be safe to lick a sharp axe. Using your better judgement, you decide against it.You can't do that.You cut the wood into smaller pieces.You can not cut that.You chop down some of the vines, which you take.You can't cut that.You can't cut that.What would you like to use it on?There are several pieces of cut wood here.There are several large pieces of wood here.drift woodThey are simply cut pieces of driftwood.They are large pieces of driftwood.They would be too awkward to walk around with.Please state what you would like to tie it with.You can't do that.You tie the wood together using the vines. A small raft is now located here.You have nothing to tie them with.You can't tie the vines with that.The wood is too large to tie up.It is already cut.What would you like to cut it with?You can't do that.It is already cut.You cut the wood into smaller pieces.You do not have an axe.You can not cut the wood with that.wooden raftThe wood is too large to use.You tie the wood and create a small raft.You use the mud to create a stronger bond between the vines and the wood.You don't have enough equipment.There is a raft here.simple wooden raftIt looks a bit unsturdy, but it may do the job.The raft is too large to carry.You can't do that.You can't put the raft in that.There is a small rock here.small brown rockYou swallow the rock in order to get your minerals for the day. Almost immediately after hitting your stomach you feel an urge to vomit, and you do so. The stone pops up and you put it back into your pocket.You must really be hungry to have an urge to eat rocks. Unfortunately, none are currently in your possesion.You can't throw that.You throw the stone over the nearby bushes. You hear a rumble as you see a large dinosaur move toward the thrown rock.You have no need to throw it at that.There is no need to throw the rock here.What would you like to throw it at?You are not holding a rock.time machineThe time machine is in generally good working order. Several of the manual controls are not present, but the Professor should hopefully be able to fix them. That is, if you ever find him.There appears to be an greasy substance around the control panel.It is currently overheated and unsafe to use. You better wait a little while.greasy substanceIt looks unpleasant.You can't do that.You already have it.You scoop out some of the substance onto a leaf.You do not have anything to hold it with.You already have it.Please state how you will pick it up.What would you like to use it on?You have no need to do that.You scoop out some of the substance onto a leaf.You do not have a leaf.There is no need to do that.You put the oily substance on the hinges of the box.You have no need to do that.large green leafYou would rather keep the leaf.You do not have it.The leaf is full of grease.It is a large green leaf.This leaf will never grow again.ClearingsYou decide that it is too dangerous for Clarisse and quickly position yourself in the machine. You nervously start the controls and hope for the best. Tha machine begins to shake and you notice a strange grinding sound eminating from the machine. You also notice that your field of view appears to be shrinking, and your vision is getting blurry. You close your eyes for a few seconds and after opening them you find yourself in a small clearing. You are still sitting inside the time machine. After regaining your composure you decide to have a look about.There appears to be a trail leading to the north and to the east. Towards the north it seems to be rocky and to the east you think you hear some type of river. You can see large tracks to the east. Other than this trail there seems to be no other exits. You are surrounded by forests. $n The time machine is located here.TrailYou are heading north on a trail leading to some hills and mountains. The view is very lovely. You look about the area, but you do not see any animal life nearby. You wonder where the Professor could be. If he took this path you would have noticed footprints. He could have wandered into the forest or took the other path. $n $n You can see a trail leading to a small opening in the trees.ForestYou are standing in the jungle. Little light is shining through the leaves and the air is very thick. A trail leads to the east and west.HillYou are standing in front of a large hill. There is a small cave within sight. You can see a path leading to the top of the hill. There is less plant life located here.HilltopYou are standing on top of the hill. You can see smoke in the far distance to your southeast. Since it doesn't appear cloudy you doubt lightning stuck and it doesn't seem a likely place for a brush fire. You think it may be the Professor. To the east is a small protruding piece of rock. You could get a better view if you could get on it. However, it does look a bit dangerous.LedgeYou try to carefully step on the piece of rock. It is a bit unstable. You can see much better from here and are almost positive it is a signal from the Professor. Too bad it won't do you much good. $n $n Just as you were trying to get back on safe ground the rock broke. You fell quite a distance. At least you died quickly...CaveYou are standing in a small, damp cave. To the east is a dark tunnel.It is too dark to safely travel in that direction.TunnelYou are standing in a small tunnel. The tunnel is quite short and ends abruptly. The ground is quite sticky with a strange mud-like substance. The only exit is to the west.DeadtunnelYou stumble around trying to see where you are going. Carefully you manage to step over the rock in front of you. Unfortunately, you hit your head on a large piece of rock overhead. You fall to the ground and die.RiverYou are standing on a dirt path. Directly before you is a waterfall that is moving very fast. At the bottom of the falls you can make out a river with a long coastline. $n $n There are also several large tracks to the north leading behind some trees and bushes. BushesYou follow the tracks to some nearby bushes. As you do so, you hear a strange sound from the other side. You can see the large head of a Tyranosaurus Rex peeking over the branches. Your scientific interest overcomes your fear as view this creature. He has slaughtered a small dinosaur and is in the process of devouring it. Exits lead north and south.RivercrossingYou take a large, flat piece of wood and try to go down the falls. As you head down the falls, you are certain you will drown. Miraculously you manage to hang onto the wood and stay afloat. You begin to float down the river when the large head of some sort of dinosaur starts to rise out of the water. You stand up on the piece of wood with the hope of diving into the water and escaping. Unfortunately, you do not make it as the creature slowly eats you.EatenYou watch as this fascinating creature walks toward you. You see his magnificent features clearly. You watch as this incredible creature swallows you whole...DrowningYou push the raft out onto the water and hope for the best. You quickly go over the falls, and as you do so, you feel the raft begin to break apart. You desparately cling to a large piece of wood, but it is of no avail. The current pulls you under, and you drown...LostYou travel down the falls on your raft. It holds together surprisingly well. The river takes you downstream farther than you had wished. You are not sure where the Professor could be, so you remain on the raft hoping to spot him on the shore. Unfortunately, by waiting so long, the raft begins to break up. You struggle to stay afloat, but it is of no use. You drown...ShoreYou travel down the falls on your raft. It holds together surprisingly well. The river takes you downstream farther than you had wished. Your raft is heading for a a number of dangerous rocks along the coastline. You dive off of your raft, just as it smashes into the rocks. Your possessions fall into the river. You manage to swim ashore, a moderate distance downstream. You look around the area as you walk several feet up the shore. It looks quite foreboding. It seems you can either follow the shoreline to the north, or go into the jungle to the east.You are standing at the shore. Exits lead north and east.Shore2You are standing on a clear shoreline. The white sand glistens as the waves slowly roll over its surface. Exits lead north and south.ShorelineYou are standing on the shoreline. There are several large dinosaurs walking about here. They appear to have some type of duck bill. A herd of these creatures are currently blocking your path. To the east is a dark path into the jungle. To the south is the shoreline.Win_gameYou drop the eggs close to the dinosaurs. The herd of dinosaurs quickly approach the eggs and begin to devour them. You make the most of this opportunity and quickly run past the creatures to the shoreline. You manage to get back on your path heading towards the fire. You walk until you are within close sight of the smoke. You think you spot a figure lying on the groundand you run to investigate. Several feet from the fire you see the Professor. He is lying on the ground, apparently hurt. He looks both surprised and happy when he sees you. $n $n "Thank God, you found me!", he exclaims. "I know I'm an old fool for trying this experiment. Clarisse must have been terribly worried." $n $n You look over the Professor to make sure he is okay. He has a few scrapes and bruises, but is generally in good shape. He tells you what happened, how the time machine malfunctioned and brought him to this time. As he was exploring the area, he got lost. He set up camp here, on the beach hoping you would come and find him. $n $n You then proceed to tell him what happened to Dr. Evers and how he is a suspect in the murder. He is shocked that Professor Evers was killed. $n $n "You know that Dr. Evers and I had our disagreements, but you know I would never do anything to harm him." "Of course not.", you answer. $n $n You both start off looking for a way back to the time machine. The Professor had been working on several components to the time machine, hoping to fix them. He says they may be able to take us back to the present time. You manage to follow a back way around the river back to the clearing where the time machine is located. The Professor installs the time components and you both sit down in the machine. As you start the controls you say a short prayer and hope you will return home. The machine begins to shake and you black out for a few seconds before you arrive back in the lab. The control panel begins to short circuit as you reach your destination. You see Clarisse running towards you with a wide smile on her face. She grabs her father and gives him a big hug. $n $n "I'm so glad you are okay!", she exclaims. $n $n As you exit the time machine, you hear someone in the next room walking towards the lab. Several policemen enter the room and proceed to arrest the Professor. They say they have found evidence linking Professor Adams to the murder. They take out a small test tube with a sample of dried blood. $n $n "This is the blood of Dr. Evers. We found a small amount of dried blood in your sink. You tried to wash off the evidence of your crime, Professor, but it didn't work.", a burly police man states in a gruff voice. $n $n The Professor looks shocked, and says he has no idea where the blood came from. $n $n Just then, a teary eyed Clarisse walks up to the police and says she killed Dr. Evers. $n $n At first the Professor and the police think she is trying to take the blame for her father. However, she eventually convinces them of her guilt as she goes into her room and comes back with a small gun. She then describes the murder in detail. $n $n "I sneaked into Dr. Evers lab to sabatoge his time travelling machine. After what he has done to father, I felt it wasn't fair if he received the government funding. As I was looking at his machine, Dr. Evers walked into the lab. He was surprised at first, and then he became furious. He said he was going to call the police and have me arrested. He also said that the government would now give him the funding, after this incident became public. I tried to escape, but he grabbed me, trying to push me to the ground. $n $n The gun accidently went off, killing him." $n $n All of you are completely shocked as she tells her story. You can see how the Professor is hurt by this news. The police take Clarisse away, and you and the Professor follow to the police station. After a trial, Clarisse is sentenced to 15 years imprisonment for manslaughter. Her father is broken hearted and enters into a lethargic state. You wish you could do something to help them, as they are both close friends. If there was some way you could change what has transpired. $n $n Then you remember. You do have a time machine. $n $n But that is another story... $n $n Congratulations, you have finished the game.Jungle4xThe jungle is quite dark and foreboding. A large pit is located here, blocking further passage into the jungle.There is no safe way down the pit.Jungle2xYou are in the jungle. It is very gloomy. A small path to the east leads further into the jungle. To the west is the shore.Jungle_pitYou are standing before a large pit blocking passage to the east. Several large trees are standing here. Exits lead west and northwest.There is no safe way down the pit.TreetopYou are located in a tall tree. Long branches spread out in several directions.BranchYou are situated on top of a long tree branch. You are directly over the large pit below.Jungle3xYou continue on into the jungle. You do not see any tracks. You stumble upon what looks like a nest. There are several very large eggs located there. To your left are a few large bushes, in which you hear strange sounds eminating. The pit is located directly behind you. The only exit is to the northwest.Jungle5xYou are standing in a relatively clear section of this jungle. You hear a few strange sounds in the distance. Exits lead southeast and west.Jungle6xYou are standing before a few large dinosaurs walking about the jungle. They have strange beaks and seem occupied looking for food. You can spot the shore through a break in the plant growth, and see several other dinosaurs gathered there.SwampYou are standing in a swampy area. Vines are hanging down from the trees. The air is very humid and moist. A large tree is standing before you.TreeYou are located up in a tree. You can see down around you for quite a distance. Swampy land surrounds this area blocking further travel. You can spot a hill to your Northeast. Leaves and vines block a full view of the surrounding lands.HiddenKilledNowhereHeroHuh?I don't understand.I don't know what you mean by 'all'.I don't know what you mean by 'it'.I don't know what you mean by 'them'.You can't refer to multiple objects with '$v'.I can't guess what you want to $v.You must supply a noun.You must give an object after 'but'.You can only use 'but' after 'all'.That doesn't leave much to $v!I don't know which $1 you mean.I can't see any $1 here.You can't go that way.You can't do that.You can't $v the $1.There is nothing here that you can $v.There is a, aand ahere.is here.Thecontains a, aand a$$.Theis empty.You have scoredpoints out ofI don't know that word.<More>(again)Enter file name to save inThat file already exists, overwrite (y) ? Sorry, save failed.Sorry, could not open the save file.Sorry, the save file was created by a different version.Sorry, the save file did not contain a save for this adventure.Enter file name to restore fromDo you want to restart (y) ?  A Matter of Time $n $n An Introductory Adventure $n $n (C) 1995 Michael Zerbo $n Created using ALAN Version 2.5 $n $n You are a research physicist named Jonathan Stone. You are standing in the Professor's laboratory. It is the year 2150. You are about to embark on a great journey, a journey into the unknown. In front of you is the Professor's time machine. The control panel is apparently missing. However, the dials are locked on a specific date. A date seventy-five million years into the past. This will be your destination, searching for your mentor and friend... $n $n Your story began several days ago as Dr. Joseph Adams and Dr. Karl Evers, bitter rivals, were each working on their version of a machine which would travel in time. They were working towards a deadline for government funding. Only one would receive it, the other left out in the cold. You worked for Professor Adams as an assistant. His daughter Clarisse also helped out when she could. $n $n You knew Professor Adams was coming close to a breakthrough with his work, although you had no idea how close he was to actually creating a working time machine. $n $n This morning Dr. Evers was found dead in his laboratory, apparently murdered. Two bullet holes were found in his head. The police visited you a short while ago, requesting the presence of Dr. Adams for questioning. To your surprise he was nowhere to be found. The police automatically began a hunt for Dr. Adams, who became their prime suspect. Just now, as you carefully look over the Professor's laboratory do you discover where the Professor went... $n $n The time machine is missing... $n $n $n You begin a search for any information the Professor may have left behind. A small note sticking out from one of the Professor's notebooks, catches your eye. You quickly remove it. $n $n $n It reads as follows: $n $n I am going alone. It is too dangerous to risk other lives. I will attempt to go as far back as we had planned, to the brink of oblivion, when time began. I will search for what man has searched for his entire existance, the proof of God. I will return on the day you are reading this note. $n $n - Professor Adams $n $n You call Clarisse into the room and show her what you found. You both can not believe that the Professor was involved in the murder. It must be just a coincidence that the Professor tested his machine this morning. You did not think the machine was ready for use, but obviously you had been mistaken. $n $n Just then, with a flash of light, the time machine reappears in the laboratory. However, it is empty. You remember a safety device inside the machine which could be set to return it to the present time in case of a problem. Unfortunately, that is what must have happened. $n $n "I am going after him", you hear Clarisse saying. You know you will have to find him if he is still alive. However, you are not sure about bringing Clarisse along. Also, you know you must hurry.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    