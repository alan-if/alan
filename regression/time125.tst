../convert/c25to26 time125 time126
../convert/c26to27 time126 time127
arun -n time127 < time125.input
rm time12[67].acd time12[67].dat
