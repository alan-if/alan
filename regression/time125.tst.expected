########## time125 ##########
../convert/c25to26 time125 time126
../convert/c26to27 time126 time127
arun time127 < time125.input
rm time12[67].acd time12[67].dat
########## /bin/sh  time125.tst ##########


As you enter the twilight zone of Adventures, you stumble and fall
to your knees. In front of you, you can vaguely see the outlines of
an Adventure that never was.

SYSTEM ERROR: Incompatible version of ACODE program. 



